Firmware/Marlin_main.cpp

```diff
@@ -3529,10 +3529,7 @@ static void gcode_M600(bool automatic, float x_position, float y_position, float
     float HotendTempBckp = degTargetHotend(active_extruder);
     int fanSpeedBckp = fanSpeed;
 
-    lastpos[X_AXIS] = current_position[X_AXIS];
-    lastpos[Y_AXIS] = current_position[Y_AXIS];
-    lastpos[Z_AXIS] = current_position[Z_AXIS];
-    lastpos[E_AXIS] = current_position[E_AXIS];
+    memcpy(lastpos, current_position, sizeof(lastpos));
```

Comment:
interesting - that means the compiler emitted a call to memcpy or some kind of loop instead of copying each float separately.